cms/urls.py

```diff
@@ -49,7 +49,7 @@
 # oauth2_urlpatterns needs to be first to override any other login and
 # logout related views.
 urlpatterns = oauth2_urlpatterns + [
-    path('', include('openedx.core.djangoapps.user_authn.urls_common')),
+    path('', include('openedx.core.djangoapps.user_authn.urls')),
```

Comment:
@syedsajjadkazmii (@connorhaugh): I had missed the significance of this line, as it relates to this comment: https://github.com/openedx/edx-platform/pull/34396#issuecomment-2017236967

My understanding of this TODO that we removed:
```
# TODO move contents of urls_common here once CMS no longer has its own login
```
was that we could get rid of urls_common from CMS, and move its urls to an app that is only used from LMS. I thought we'd have less views in CMS, not more. I just missed this particular change, that made things go in the wrong direction.

Did I miss the boat? Or would we want to undo this change, and maybe just start by deleting this line instead?

Sorry if I am just adding confusion, and totally missed the point of the clean-up.

Reply:
Hi @robrap thanks for pointing this out.

I just checked New Relic and found the usage of user_authn Logout and Register views. (Attaching SS)
It seems like these two URLs are in use by CMS. I am not a New Relic expert so I don't know how to verify if these are bots. Do you suggest reverting this change or should I just delete the line above ^.

<img width="1440" alt="Screenshot 2024-03-26 at 11 14 28 AM" src="https://github.com/openedx/edx-platform/assets/52817156/291febd0-ab0a-44a5-ad2d-bdddb3351ee1">

Note: The date range I've selected ends before the date of this PR deployment.

Reply:
- I'm sorry to say this, but would you mind reverting until we work this out? By adding even more URLs to cms that we don't want used by the cms, we only compound this problem if we later find traffic and are unsure if it is real or not.
- Some follow-up ideas:
  - Investigating whether this traffic is real or not, and figuring out if it should be real, or if certain UI is sending users to the wrong place. I could help with that, but this is not the best time. You could also use Slack observability channel for support, or defer this work.
  - Separately, if there are URLs we know are only used by lms, and not cms, we could move them from common to the other file so they are simply removed from the cms.
  - Lastly, any changes could go behind a temporary rollout toggle so you could always enable/disable without a deploy in case anything goes wrong.

Reply:
Yes, I have reverted the change ([revert](https://github.com/openedx/edx-platform/pull/34431)). 

We have created a ticket in our backlog to further work on the follow-up ideas later.

Reply:
Thank you. Sorry I didn’t notice this in the original review.